aptos-move/block-executor/src/executor.rs

```diff
@@ -271,27 +228,35 @@ where
             scheduler.finish_abort(idx_to_validate, incarnation)
         } else {
             scheduler.finish_validation(idx_to_validate, validation_wave);
+
+            if valid {
+                scheduler.queueing_commits_arm();
+            }
+
             SchedulerTask::NoTask
         }
     }
 
-    fn coordinator_commit_hook(
+    fn queue_ready_commits(
```

Comment:
for the naming here, this does do some work, and with aggregators_v2 will do even more work (validate is speculation was correct, and if not, re-execute transaction inline, to produce the final output)

maybe calling it prepare_and_queue_ready_commits ? (maybe even prepare_and_queue_for_materialization_ready_commits  :) )

Reply:
@igor-aptos I hope we'll not bloat this function with complicated functionality, as this is the one executed serially.

About the name, how about `prepare_and_queue_commit_ready_txns`